frontend/src/app/reducers/varianttests.js

```diff
@@ -38,9 +38,34 @@ const tests = [
 const chosenVariants = {};
 const identityReducers = {};
 
+let chosenTest = null;
+
 tests.forEach(test => {
-  chosenVariants[test.name] = test.getValue();
+  // Only put each user in one test. If we previously found a non-default test,
+  // put this user in default for the rest of the tests.
```

Comment:
So, for example, if there are three tests, the first test should have a 1/3 chance of getting a user, and a 2/3 chance of default, and then the second test should have 1/2 and 1/2.